releases/release-7.2.0.md

```diff
@@ -0,0 +1,353 @@
+---
+title: TiDB 7.2.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.2.0.
+---
+
+# TiDB 7.2.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.2.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.2/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.2.0#version-list)
+
+7.2.0 introduces the following key features and improvements:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">Scalability and Performance</td>
+    <td>Resource groups support managing runaway queries (experimental)</td>
+    <td>Resource groups can manage queries that take longer time than expected by either deprioritizing or killing them. This feature enables you to set the maximum execution time for queries at the resource group level and specify the actions to be taken by TiDB when a query exceeds this limit, which offers you greater control over the impact of unexpectedly long-running queries on your cluster.
+    </td>
+  </tr>
+  <tr>
+    <td>TiFlash supports the pipeline execution model (experimental)</td>
+    <td>TiFlash supports a pipeline execution model to optimize thread resource control.
+This is currently depended on by TiDB Serverless analytical capabilities.</td>
+  </tr>
+  <tr>
+    <td rowspan="1">SQL</td>
+    <td>Support a new SQL statement, `IMPORT INTO`, to enable data import using the TiDB service, itself (experimental)</td>
+   <td>To simplify the deployment and maintenance of TiDB Lightning, TiDB introduces a new SQL statement `IMPORT INTO`, which integrates physical import capabilities of TiDB Lightning, including remote import from Amazon S3 directly into TiDB.</td>
+  </tr>
+  <tr>
+    <td rowspan="2">DB Operations and Observability</td>
+    <td>DDL supports pause and resume operations (experimental)</td>
+    <td>DDL operations such as adding indexes can consume a lot of resources and affect online traffic. As homage to traditional schema change tools like pt-osc and gh-ost, TiDB now natively supports suspending multiple background jobs at once. This feature lets you conserve resources and minimize the impact on online traffic, without the need to cancel and restart the jobs.</td>
```

Comment:
@samhld , For "As homage to", do we have any alternative way to express the same meaning?

Reply:
@qiancai the former. "Inspired by".

Reply:
@benmaoer PTAL

Reply:
Introducing the DDL Pause and Resume feature! With this new capability, users can now temporarily suspend resource-intensive DDL operations, such as index creation, to conserve resources and minimize impact on online traffic. Seamlessly resume these operations when ready, without the need to cancel and restart. Enhance resource utilization, improve user experience, and streamline schema changes with DDL Pause and Resume.